public/css/secondary.css

```diff
@@ -1,38 +0,0 @@
-header {
```

Comment:
I was able to add this to the bottom of core.scss, since it is imported on every page core.scss is imported on.